pyro/contrib/oed/eig.py

```diff
@@ -6,10 +6,11 @@
 from pyro.contrib.oed.search import Search
 from pyro.infer import EmpiricalMarginal, Importance, SVI
 from pyro.contrib.autoguide import mean_field_guide_entropy
+from pyro.contrib.oed.util import lexpand
 
 
-def vi_ape(model, design, observation_labels, vi_parameters, is_parameters,
-           y_dist=None):
+def vi_ape(model, design, observation_labels, target_labels,
+           vi_parameters, is_parameters, y_dist=None):
```

Comment:
The common API for eig estimators is `eig(model, design, observation_labels, target_labels, ...)`